api/src/main/java/jakarta/xml/bind/ServiceLoaderUtil.java

```diff
@@ -74,6 +74,25 @@ static Object lookupUsingOSGiServiceLoader(String factoryId, Logger logger) {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    static <T> Iterable<T> lookupsUsingOSGiServiceLoader(String factoryId, Logger logger) {
```

Comment:
hm, why exactly was it required to add this new method? Wouldn't it be better to fix existing method instead of introducing this new one?